tests/health_checks/NDv4/run-health-checks.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+check_hardware=$(./check-hardware.sh)
+detect=$(echo "$check_hardware" | grep "Fail:" | wc -l)
+
+if [ $detect -gt 0 ]; then
+	echo "Failed to detect GPUs and NICs, run check-hardware.sh"
+else
+        #run 'dcgmi diag -r 1' and check for failures
+	./dcgmi-check.sh
+        #run 'nvidia-smi -q -d ROW_REMAPPER' and check for problems 
+	./ecc-errors.sh
+        #run ibstat and check all devices are set up as expected
+	./test-ib.sh
+        #test that host to device/device to host bandwidth meet expectations.
+	./test-bandwidth.sh
+        #test nccl using nvlink and ib
+	./test-nccl.sh
+fi
```

Comment:
Is it safe to assume the scripts are always invoked under tests/health_checks/NDv4?

Reply:
This is a good point. Perhaps I should allow the user to specify a path, but otherwise default to tests/health_checks/NDv4.

Reply:
User should be able to invoke run-health-checks.sh in any path, as long as they put the correct path. 
Inside run-health-checks.sh, we just need to ensure we can find those test-*.sh (maybe use pushd and popd) and invoke them.

Reply:
Users can now pass a directory to the run-health-checks.sh script, or it will just assume the health checks are in the default location. It will run wherever you execute from.